### ISSUE TITLE
[QUESTION] Work on the custom powershell file delivery?

### ISSUE THREAD

**Issue by thel3l** [offensive-security/nethunter-app#144]
Hey.

@binkybear in another commit in nethunter-utils , you mentioned that you intended to set up a custom Powershell upload technique. I've started work on this (not yet committed), but I'd like to know if this is something that I should continue working on.

Thanks.

**Comment by jmingov** [comment]
Hi @thel3l , can you provide some more detail of the implementation? Maybe push the changes to a cloned repo or something like that.

BR

J

**Comment by thel3l** [comment]
@jmingov Yep. Sorry about the lack of details. But the second pull request is the one I was referencing. 

I'm currently on a mobile. If you're interested, I can just push the changes I was making to my repo and open a PR. I wasn't sure as in the last PR on nethunter-utils, @binkybear said he'd have to test it out a bit and then decide if this functionality was required.

**Comment by thel3l** [comment]
@jmingov Take a look at [PR#145](https://github.com/offensive-security/nethunter-app/pull/145) . It's what I pieced together.
